src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.Subscription.cs

```diff
@@ -58,7 +60,7 @@ private sealed class Subscription : ISubscription
             /// in that compilation. We use this property as a proxy for compilation start time, whereas
             /// the outputs represent compilation end time.
             /// </remarks>
-            private DateTime _lastCheckedAtUtc = DateTime.MinValue;
+            private readonly Dictionary<ProjectConfiguration, DateTime> _lastCheckedAtUtc = new();
```

Comment:
The name `_lastCheckedAtUtcByConfiguration` would be more consistent with other dictionary names in this area of the code. 

This convention makes the variable sound like a dictionary. The current name sounds like a `DateTime`.